src/test/kotlin/io/meshcloud/dockerosb/metrics/GaugeMetricsServiceTest.kt

```diff
@@ -0,0 +1,81 @@
+package io.meshcloud.dockerosb.metrics
+
+import io.meshcloud.dockerosb.metrics.gauge.GaugeMetricProvider
+import io.meshcloud.dockerosb.metrics.gauge.PagedGaugeController
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.springframework.cloud.servicebroker.model.catalog.Catalog
+import java.time.Instant
+
+class GaugeMetricsServiceTest : BaseMetricsServiceTest() {
+
+  private val serviceDefinitionId = "d40133dd-8373-4c25-8014-fde98f38a728"
+  private val serviceInstanceId = "testInstanceID"
+  private val serviceInstanceId2 = "testInstanceID2"
+  private val resource1 = "test"
+  private val resource2 = "testSecond"
+  private val firstStartDate = Instant.parse("2018-01-01T12:00:00Z")
+  private val secondStartDate = Instant.parse("2020-01-01T12:00:00Z")
+  private val endDate = Instant.parse("2022-12-01T12:00:00Z")
+
+  private lateinit var gaugeController:PagedGaugeController
+
+  @Before
```

Comment:
f: I think this can be declared in line 22 instead so we can remove the @Before. Its always the same anyway

Reply:
somehow it didnt work like that so i couldnt get rid of it. it says the catalogService must be initialized first etc. probably there is a way to make it work but im not sure how much i will get benefit from it. so i stopped trying 😄